pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
```

Comment:
Correct. This docstring is a copy-paste artifact (I used pip help command as a template). I'll correct that. This patch does include the docs for the rwt command, and in fact the tests ensure that `pip help run` outputs the same as `pip run --help`.